ska_builder.py

```diff
@@ -203,9 +213,49 @@ def build_list_packages(pkg_names, args, src_dir, build_dir):
         raise ValueError("Packages {} failed".format(",".join(failures)))
 
 
+def overwrite_skare3_version(current_version, new_version, pkg_path):
+    meta_file = pkg_path / 'meta.yaml'
+    with open(meta_file) as fh:
+        t = jinja2.Template(fh.read())
+    text = (t.render(SKA_PKG_VERSION='$SKA_PKG_VERSION',
+                     SKA_TOP_SRC_DIR='$SKA_TOP_SRC_DIR'))
+    if version.parse(yaml.__version__) < version.parse("5.1"):
+        data = yaml.load(text)
+    else:
+        data = yaml.load(text, Loader=yaml.BaseLoader)
+    if str(data['package']['version']) == str(current_version):
+        data['package']['version'] = new_version
+        for i in range(len(data['requirements'])):
+            if re.search('==', data['requirements']['run'][i]):
+                name, pkg_version = data['requirements']['run'][i].split('==')
+                name = name.strip()
+                if re.match('ska3-\S+$', name) and pkg_version == current_version:
+                    data['requirements']['run'][i] = f'{name} =={new_version}'
+        t = string.Template(yaml.dump(data, indent=4)).substitute(
+            SKA_PKG_VERSION='{{ SKA_PKG_VERSION }}',
+            SKA_TOP_SRC_DIR='{{ SKA_TOP_SRC_DIR }}'
+        )
+        with open(meta_file, 'w') as f:
+            f.write(t)
+    else:
+        print(f'  - NOT changing version on {pkg_path}')
+
+
 def main():
     args = get_opt()
 
+    if args.skare3_overwrite_version:
+        if ':' not in args.skare3_overwrite_version:
```

Comment:
Add a comment about the intent of this block of code. It is entirely not obvious without reading the PR description.

Reply:
Done.